templates/common/.config/webpack/utils.ts

```diff
@@ -15,3 +19,23 @@ export function getPluginId() {
 export function hasReadme() {
   return fs.existsSync(path.resolve(process.cwd(), SOURCE_DIR, 'README.md'));
 }
+
+export async function getEntries(): Promise<Record<string, string>> {
+  const parent = `..${path.sep}`;
```

Comment:
I don't think there's a need for a separator as `path.resolve` and `path.join` should handle that internally.